src/controllers/base.js

```diff
@@ -25,6 +25,9 @@ export function toFixed(value) {
   return Number.parseFloat(value).toFixed(decimals);
 }
 
+const configKeys = ['outlierRadius', 'itemRadius', 'itemStyle', 'itemBackgroundColor', 'itemBorderColor', 'outlierColor', 'medianColor', 'hitPadding', 'outlierHitRadius'];
```

Comment:
with this change we need to have the `medianColor` defined in the datasets, right? at least the `vertical.html` sample throws errors if I don't have the `medianColor` defined and try to hover over the chart

should we provide any fallback? not sure how this usually works in Chart.js

Reply:
should be fixed